### PR TITLE
Modifies the documentation to instruct users to use the latest tagged version

### PR DESCRIPTION
**Pull request by PrahlM93**
Modifies the documentation to instruct users to use the latest tagged version

### CODE REVIEW

**Comment by PrahlM93** [comment]
Addresses #141

File: docs/Installation/Docker.md [hunk 1]
```diff
@@ -27,22 +27,30 @@ bind-address is set 0.0.0.0 and not 127.0.0.1 in:
...
+1. Find the URL to the latest tar.gz file containing the PostMaster source code at:
 
-        git clone https://github.com/StackFocus/PostMaster.git ~/postmaster
+        https://github.com/StackFocus/PostMaster/releases/latest
 
-2. Build the Docker image:
+2. Then download PostMaster with the URL from the previous step:
 
-        cd ~/postmaster
+        wget https://url.from.step.one -O ~/postmaster.tar.gz
```
Comment (thatarchguy) [inline docs/Installation/Docker.md]: Hm...maybe use an example link instead?
`wget https://github.com/StackFocus/PostMaster/archive/v1.0.0.0.tar.gz` or 
`wget https://github.com/StackFocus/PostMaster/archive/[release version].tar.gz`

Comment (PrahlM93) [inline docs/Installation/Docker.md]: Yeah I like the second idea. I'll change it tonight.

Comment (PrahlM93) [inline docs/Installation/Docker.md]: Fixed this in the latest rebase.

**Pull request merged**